typescript/widgets/src/chains/ChainSearchMenu.tsx

```diff
@@ -150,164 +144,164 @@ export function ChainSearchMenu({
   );
 }
 
-function ChainListItem({
-  data: chain,
-  customField,
-}: {
-  data: ChainMetadata;
-  customField?: CustomListItemField | null;
-}) {
-  return (
-    <>
-      <div className="htw-flex htw-items-center">
-        <div className="htw-shrink-0">
-          <ChainLogo chainName={chain.name} logoUri={chain.logoURI} size={32} />
-        </div>
-        <div className="htw-ml-3 htw-text-left htw-overflow-hidden">
-          <div className="htw-text-sm htw-font-medium truncate">
-            {chain.displayName}
+const ChainListItem = memo(
```

Comment:
Leave this as a `function` component and memoize it below, like we do here: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/widgets/src/icons/Wallet.tsx#L7